src/data_info.py

```diff
@@ -93,11 +93,8 @@ def write_email_template(
         df: pd.DataFrame,
         dataframe_saver: DataFrameSaver,
     ) -> None:
-        try:
-            emails = cls.get_emails(df=df)
-        except Exception as e:
-            print(e)
-            emails = pd.Series()
```

Comment:
this exception area wasn't needed? I don't remember why @guillermoojeda was added, but I think it was a bug in the code.

Reply:
I've just realized that you mentioned this in the description of the PR. sorry, I close this thread